### PR TITLE
usnic: Fixes to the CM handling

### PR DESCRIPTION
**Pull request by bturrubiates**
This set of commits fixes several problems with the CM data handling. 

@goodell @jsquyres

### CODE REVIEW

File: prov/usnic/src/usdf_eq.c [hunk 1]
```diff
@@ -93,6 +93,13 @@ static inline ssize_t usdf_eq_read_event(struct usdf_eq *eq, uint32_t *event,
 
 	copylen = MIN(ev->ue_len, len);
 
+	if (copylen < ev->ue_len) {
+		USDF_DBG_SYS(EP_CTRL,
```
Comment (goodell) [inline prov/usnic/src/usdf_eq.c]: I don't feel strongly about this, but should this be a `USDF_WARN_SYS` instead?

**Comment by goodell** [comment]
👍

File: prov/usnic/src/usdf_eq.c (hunk 1 above)
Comment (bturrubiates) [inline prov/usnic/src/usdf_eq.c]: > I don't feel strongly about this, but should this be a USDF_WARN_SYS instead?

Sure

**Comment by jsquyres** [comment]
👍

**Pull request merged**